src/Compilers/CSharp/Portable/Binder/BindingDiagnosticBag.cs

```diff
@@ -16,21 +16,23 @@ internal sealed class BindingDiagnosticBag : BindingDiagnosticBag<AssemblySymbol
     {
         public static readonly BindingDiagnosticBag Discarded = new BindingDiagnosticBag(null, null);
 
+        private static readonly Func<DiagnosticInfo, DiagnosticBag, Location, bool> s_reportUseSiteDiagnostic = Symbol.ReportUseSiteDiagnostic;
```

Comment:
Curious: do we need this pattern anymore? The compiler automatically caches method group conversions now.